testcafe/e2e/changeLanguage.ts

```diff
@@ -8,9 +10,16 @@ fixture('Landing page').page(getEnvUrl('/fi/home'));
 
 test('Changing language on landing page', async (t) => {
   await t
-    .expect(screen.getAllByRole('link', { name: /etsi tekemistä/i }).count)
+    .expect(
+      screen.getAllByRole('link', { name: translationsFi.header.searchEvents })
+        .count
+    )
     .eql(2)
```

Comment:
expect(...).toHaveLength(); could be used here also. But this is minor, no need to change if you feel that .eql() is better

Reply:
Actually not, because it uses testcafe assertion. But I improved the test to check links both within footer and header separately.

Reply:
Sorry, didn't event notice that this is actually testcafe test. my bad.